### DISCUSSION TITLE
N501Y

### DISCUSSION THREAD

**Discussion by jank3** [brianhie/viral-mutation#38] [Q&A]
Thanks for share your works! 

But one thing, the prediction of N501 to Y is very common in SARS-CoV-2 variants but is not consider in the semantic COV2 results, please could you tell me why?

Thanks in advance

**Comment by brianhie** [comment]
Thanks for the question, I think the coronavirus model was trained very early in the pandemic with a low number of sequences, so it probably did not pick up N501Y. Since then, there have been better models trained such as https://www.science.org/doi/full/10.1126/scitranslmed.abk3445 and https://www.biorxiv.org/content/10.1101/2022.07.21.501023v1.full
